app/screens/gallery/gallery.test.js.disabled

```diff
@@ -2,11 +2,11 @@
 // See LICENSE.txt for license information.
 
 import React from 'react';
-import {shallowWithIntl} from 'test/intl-test-helper';
+import {shallowWithIntl} from '@test/intl-test-helper';
 
 import Preferences from '@mm-redux/constants/preferences';
 
-import * as NavigationActions from 'app/actions/navigation';
+import * as NavigationActions from '@actions/navigation';
```

Comment:
Shouldn't the NavigationActions import be above the shallowWithIntl line ?

Correct order should be `NavigationActions`, `shallowWithIntl` and Preferences .

Reply:
My guess is that the linter did not go through it because it is a `.disabled` extension, and not a `.js`. I can fix it manually, or we can leave it to get fixed when we remove the `.disabled`.